src/lib/containers/entity_finder/tree/TreeView.tsx

```diff
@@ -236,7 +243,7 @@ export const TreeView: React.FunctionComponent<TreeViewProps> = ({
                   scopeOption === FinderScope.CURRENT_PROJECT &&
                   initialContainerId == null
                 ) {
-                  return <></>
+                  return <React.Fragment key={scopeOption} />
```

Comment:
I bet there's a story to be told here!

Reply:
I don't think it would have caused any issues since this doesn't really get re-rendered. It gets rid of annoying red text in the console, though!